code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -134,3 +144,54 @@ func (edb *EventDb) GetTransactionsForBlocks(blockStart, blockEnd int64) ([]Tran
 		Find(&tr)
 	return tr, res.Error
 }
+
+func (edb *EventDb) UpdateTransactionErrors() error {
+
+	var transactions []Transaction
+
+	// created_at for last day from now
+	lastDay := time.Now().AddDate(0, 0, -1)
+	// convert to string
+	lastDayString := lastDay.Format("2006-01-02 15:04:05")
+
+	logging.Logger.Info("lastDayString", zap.Any("lastDayString", lastDayString), zap.Any("lastDay", lastDay), zap.Any("time.Now()", time.Now()))
+
+	// read all the transactions from the transaction table where status is 2 till last day
```

Comment:
We are running this update every 60 seconds, so we don't need to get all from last day on every query, which could cause performance issue. So perhaps from last 2 minutes is enough.